cree el archivo raul.py

raudev-h opened the pull request:
Revisa los cambios

raudev-h committed:
cree el archivo raul.py

printaniel reviewed (changes requested):
Cambia la salida del print a "Hola Mundo"...sin !!

raudev-h committed:
Update raul.py

Ya implenté lo que me pediste y agregue una nueva funcion que pienso que mejora aún más el código
